Gemfile

```diff
@@ -1,2 +1,5 @@
 source 'https://rubygems.org'
 gem 'github-pages'
+group :jekyll_plugins do
+  gem 'us_web_design_standards'
```

Comment:
you may want to check that the assets work correctly for things like alerts.

There's a branch that fixes it, but it's not merged in yet because of some jekyll issues.
https://github.com/18F/us_web_design_standards_gem/pull/7

We're currently pulling in the branch in caseflow:
https://github.com/department-of-veterans-affairs/caseflow-efolder/blob/master/Gemfile#L20